releases/v21.1.md

```diff
@@ -0,0 +1,3293 @@
+[#65358]: https://github.com/cockroachdb/cockroach/pull/65358
+[#65359]: https://github.com/cockroachdb/cockroach/pull/65359
+[#65361]: https://github.com/cockroachdb/cockroach/pull/65361
+[#65363]: https://github.com/cockroachdb/cockroach/pull/65363
+[#65368]: https://github.com/cockroachdb/cockroach/pull/65368
+[#65391]: https://github.com/cockroachdb/cockroach/pull/65391
+[#65428]: https://github.com/cockroachdb/cockroach/pull/65428
+[#65442]: https://github.com/cockroachdb/cockroach/pull/65442
+
+## v21.1.0
+
+With the release of CockroachDB v21.1, we've made a variety of flexibility, performance, and compatibility improvements. Check out a [summary of the most significant user-facing changes](#feature-summary) and then [upgrade to CockroachDB v21.1](../v21.1/upgrade-cockroach-version.html).
+
+To learn more:
+
+- Read the [v21.1 blog post](https://www.cockroachlabs.com/blog/cockroachdb-21-1-release).
+- Join us on May 19 for a [livestream on why multi-region applications](https://www.cockroachlabs.com/webinars/the-cockroach-hour-multi-region/) matter and how our Product and Engineering teams partnered to make them simple in v21.1.
+
+### Downloads
+
+<div id="os-tabs" class="filters clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v21.1.0.linux-amd64.tgz"><button id="linux" class="filter-button" data-scope="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v21.1.0.darwin-10.9-amd64.tgz"><button id="mac" class="filter-button" data-scope="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v21.1.0.windows-6.2-amd64.zip"><button id="windows" class="filter-button" data-scope="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v21.1.0.src.tgz"><button id="source" class="filter-button" data-scope="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+<section class="filter-content" data-scope="windows">
+{% include windows_warning.md %}
+</section>
+
+### Docker image
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ docker pull cockroachdb/cockroach:v21.1.0
+~~~
+
+### CockroachCloud
+
+- <a href="https://cockroachlabs.cloud/signup?referralId=docs_crdb_release_notes" rel="noopener" target="_blank">Get a free v21.1 cluster on CockroachCloud</a>.
+- Learn about recent updates to CockroachCloud in the [CockroachCloud Release Notes](index-cockroachcloud.html).
+
+### Feature summary
+
+This section summarizes the most significant user-facing changes in v21.1.0. For a complete list of features and changes, including bug fixes and performance improvements, see the [release notes](index.html#testing-releases) for previous testing releases. You can also search for [what's new in v21.1 in our docs](../search.html?query=%22New%20in%20v21.1%22&page=2).
```

Comment:
This section (the hand-summarized version of the 21.1.0 notes) shouldn't be mixed in to the middle of this giant page - I'd probably keep it on its own page. Maybe three pages per branch:
- "What's new in 21.1" - the hand-summarized overall notes. (The "what's new" name is inspired by [python](https://docs.python.org/3/whatsnew/3.9.html))
- "21.1 pre-releases" - all the alpha and beta release notes
- "21.1 patch releases"  - all the 21.1.x release notes starting with 21.1.1. Alternately, call it "21.1 releases" and start it with the mostly-auto-generated form of the 21.1.0 release notes (or a merger of the alpha/beta notes). That's probably not worth the trouble, though, since there's still a lot of editing that goes into the output of the release notes script.